Sources/OpenAPIRuntime/Base/Base64EncodedData.swift

```diff
@@ -56,7 +56,23 @@ public struct Base64EncodedData: Sendable, Hashable {
 
     /// Initializes an instance of ``Base64EncodedData`` wrapping the provided slice of bytes.
     /// - Parameter data: The underlying bytes to wrap.
+    @available(*, deprecated, renamed: "init(_:)")
+
```

Comment:
```suggestion
```

Reply:
I get swift-format error if I remove the line. Only if I put them on the same line, it works.

Reply:
Ok that's fine then, thanks!

Reply:
You're welcome!!